thorlcr/graph/thgraph.cpp

```diff
@@ -2516,7 +2516,7 @@ void CJobBase::init()
     thorAllocator.setown(createThorAllocator(((memsize_t)globalMemorySize)*0x100000, memorySpillAt, *logctx, crcChecking, usePackedAllocator));
 
     unsigned defaultMemMB = globalMemorySize*3/4;
-    unsigned largeMemSize = getOptUInt("@largeMemSize", defaultMemMB);
+    unsigned largeMemSize = getOptUInt("largeMemSize", defaultMemMB);
```

Comment:
Not directly related, but correction for Debug/ , caused this to fail (i.e. was looking for @@largeMemSize)
largeMemSize is legacy anyway , I will remove related code in another PR.